src/JWTs.jl

```diff
@@ -109,10 +109,10 @@ isvalid(jwt::JWT) = jwt.valid
 
 Get the key id from the JWT header, or `nothing` if the `kid` parameter is not included in the JWT header.
 
-The JWT must be signed. An `AssertionError` is thrown otherwise.
+The JWT must be signed. An `ArgumentError` is thrown otherwise.
```

Comment:
If you want, we could make this a little more vague, by just saying "exception" instead of specifying the exception type in the public docstring. That way, if in the future you want to change the exception type, you can do so without making a breaking change (because the thrown exception type wouldn't be part of the public API).

```suggestion
The JWT must be signed. An exception is thrown otherwise.
```

Reply:
Yes, that makes sense. Thanks!